src/resourcetiming-compression.js

```diff
@@ -1063,6 +1083,25 @@
             } else {
                 results[finalUrl] = data;
             }
+
+            if (e.hasOwnProperty("_data")) {
```

Comment:
There's a `if (!e.hasOwnProperty("_data")) {` clause in our Boomerang version, that's not needed here?

Reply:
The !e.hasOwnProperty regards dimensions. As we don't have dimensions here, I didn't include that code.

Reply:
Dimensions (i.e. width x height) processing is just a few lines above